patches/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java.patch

```diff
@@ -0,0 +1,47 @@
+--- ../src-base/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java
++++ ../src-work/minecraft/net/minecraft/client/gui/advancements/GuiAdvancementTab.java
+@@ -37,7 +37,14 @@
+     private int field_191814_q = Integer.MIN_VALUE;
+     private float field_191815_r;
+     private boolean field_192992_s;
++    private int page;
+ 
++    public GuiAdvancementTab(Minecraft p_i47589_1_, GuiScreenAdvancements p_i47589_2_, AdvancementTabType p_i47589_3_, int p_i47589_4_, int page, Advancement p_i47589_5_, DisplayInfo p_i47589_6_)
```

Comment:
Please add new methods/constructors to the end of the file in a `/***** FORGE *****/` section like some other patches.